electron/js/preload.js

```diff
@@ -139,8 +139,10 @@ ipcRenderer.once('webapp-loaded', function(sender, config) {
   // else we are on /auth
   try {
     Object.assign(window.sodium, require('libsodium-neon'));
+    console.log('Using libsodium-neon.');
```

Comment:
On a second thought.. we don't use dots anywhere as far as I know.. so let's just remove it